ci/tests/puppeteer/run.sh

```diff
@@ -29,7 +29,8 @@ random=$(openssl rand -hex 8)
 
 if [[ ! -d "$PWD"/ci/tests/puppeteer/node_modules/puppeteer ]] ; then
   echo "Installing Puppeteer"
-  npm i --prefix "$PWD"/ci/tests/puppeteer puppeteer jsonwebtoken axios request
+  npm_install_cmd="npm i --prefix "$PWD"/ci/tests/puppeteer puppeteer jsonwebtoken axios request"
+  eval $npm_install_cmd || eval $npm_install_cmd || eval $npm_install_cmd
 else
```

Comment:
I saw an npm install fail due to timeout, this is just retry 3 times (or until it works, whichever happens first). Also doubled the timeout in .npmrc for the puppeteer project folder. Confirmed that it is doubled by running:
```npm config ls -l --prefix ci/tests/puppeteer/ | grep fetch-retry-maxtimeout```